APMrover2/navigation.cpp

```diff
@@ -22,7 +22,7 @@ void Rover::navigate()
 	wp_distance = get_distance(current_loc, next_WP);
 
 	if (wp_distance < 0){
-		gcs_send_text(MAV_SEVERITY_CRITICAL,"<navigate> WP error - distance < 0");
+		gcs_send_text(MAV_SEVERITY_NOTICE,"<navigate> WP error - distance < 0");
```

Comment:
How bad is it to have code that never executes? The distance between two waypoints is a scaler and is never less than 0. This is ASSERT as thing would be really bad if it was less than zero.
https://github.com/diydrones/ardupilot/blob/Copter-3.3/libraries/AP_Math/location.cpp#L61
https://github.com/diydrones/ardupilot/blob/Copter-3.3/libraries/AP_Math/AP_Math.cpp#L112